scripts/scr_ruins_reward/scr_ruins_reward.gml

```diff
@@ -138,13 +138,12 @@ function scr_ruins_reward(argument0, argument1, argument2) {
 	    var pop;pop=instance_create(0,0,obj_popup);
 	    pop.image="ruins_ship";pop.title="Ancient Ruins: Starship";
 	    pop.text="The ground beneath one of your battle brothers crumbles, and he falls a great height.  The other marines go down in pursuit- within a great chamber they find the remains of an ancient starship.  Though derelict, it is possible to land "+string(obj_ini.role[100,16])+"s onto the planet to repair the ship.  10,000 Requisition will be needed to make it operational.";
-	    argument0.p_feature[argument1]=string_replace(argument0.p_feature[argument1],"Ancient Ruins|","Starship!0!|");
+		argument0.p_feature[argument1][search_planet_features(argument0.p_feature[argument1],P_features.Ancient_Ruins)[0]].find_starship();
```

Comment:
could you rename argument0 to something more descriptive?

Reply:
replaced all variables with new names in function